components/restapi/core/component.go

```diff
@@ -39,14 +39,6 @@ const (
 	// GET returns block metadata.
 	RouteBlockMetadata = "/blocks/:" + restapipkg.ParameterBlockID + "/metadata"
 
-	// RouteBlocks is the route for creating new blocks.
-	// POST creates a single new block and returns the new block ID.
-	// The block is parsed based on the given type in the request "Content-Type" header.
-	// By providing only the protocolVersion and payload transaction user can POST a transaction.
-	// MIMEApplicationJSON => json.
-	// MIMEVendorIOTASerializer => bytes.
-	RouteBlocks = "/blocks"
```

Comment:
We still need this route, but it should pass the blocks to the blockhandler. Else there is no way for clients to send pre-signed blocks.